CONTRIBUTING.md

```diff
@@ -103,17 +103,16 @@ dnf install golang-bin git
 ```
 
 ### Developer flags
+* `--dev-package-managers` (deprecated): This flag is deprecated and currently a no-op. 
+  It will be removed in a future version. Use the new experimental syntax `x-<pkg>` instead.
+* `x-<pkg>` (experimental): enables the in-development package manager `<pkg>` for testing. Please refer to other existing package managers to see how they’re enabled and wired to the CLI.
```

Comment:
What does `(experimental)` mean here? That the feature itself is experimental or that it enables an experimental feature? @ben-alkov I skimmed through md linter config and could not find any rule that would forbit checking list items for length, what did I miss?